docs/install.md

```diff
@@ -6,19 +6,24 @@ This package may be used to setup and run an [OPTIMADE index meta-database](http
 Install the package via `pip install optimade[server]`.
 
 This python OPTIMADE implementation can be configured in two ways:
-First, the server can be configured via environment variables prefixed with `optimade_` and the corresponding variable names in `ServerConfig` in the [`config.py` file](optimade/server/config.py). These take precedence. These environment variables are not case sensitive, so both `OPTIMADE_CONFIG_FILE` and `optimade_config_file` are valid.
-Second, and the preferred method of configuring the server, is to use a [JSON file](example_json.config) with the bulk of the configuration and setting the `optimade_config_file` environment variable to point to the absolute location of this JSON file. By default this points to `~/.optimade.json` which can also be used to store the configuration if you don't want to set `optimade_config_file`.
+First, the server can be configured via environment variables prefixed with `optimade_` and the corresponding variable names in `ServerConfig` in the [`config.py` file][optimade.server.config]. These take precedence. These environment variables are not case sensitive, so both `OPTIMADE_CONFIG_FILE` and `optimade_config_file` are valid.
```

Comment:
Indeed. The weird square bracket links is a feature by `mkdocstrings` to be able to reference relative link anchors that may exist in the API Reference section.

Reply:
I think this is fine, considering a lot of other repositories do a similar thing: Favoring their docs setup rather than the GitHub rendered file...
Personally, I don't like when this is done for the README though and would instead prefer to have these be separate for the docs `index.md` and the root `README.md`, if they cannot work both places.